docker/build_support/build_image

```diff
@@ -100,6 +101,7 @@ function runBuild() {
     --cache-from $tag \
     --cache-from $baseTag \
     --build-arg APP_NAME=$baseTag \
+    --build-arg MONOETNA_SHA=$monoetnaSha \
```

Comment:
For the production / staging images, this bakes the git sha into the image directly.